Lib/json/tool.py

```diff
@@ -16,6 +16,18 @@
 import sys
 
 
+def parse_indent(indent):
+    """Parse the argparse indent argument."""
+    if indent == 'None':
+        return None
+    if indent == r'\t':
+        return '\t'
```

Comment:
@methane can you think of any other way to expose setting `indent`? The issue is that `indent` can take an int, string, or None. Command line arguments are not ideal for type flexibility. Furthermore, the most common string is a tab, which is difficult to pass via the command line without such special casing.

Another option would be to pass `options.indent` through [`ast.literal_eval`](https://docs.python.org/3/library/ast.html#ast.literal_eval). Therefore, users could specify any python value, but the command line usage would be more cumbersome.